texts/compatibility.js

```diff
@@ -119,3 +119,26 @@ texts.push([
 	"Αναζήτηση", // 18
 	"Επιλογή γλώσσας" // 19
 ])
+
+// Vietnamese (translation provided by Bentokun)
+texts.push([
+	"Không rõ", // 1
```

Comment:
Translation is messed up a bit. its missing a line.
The counting for the sentences starts from zero, so this should be fixed.

Reply:
yes, its just fine (as its comment), but a bit messed up :)